### DISCUSSION TITLE
Imported pcap not loading

### DISCUSSION THREAD

**Discussion by Dapylad** [Security-Onion-Solutions/securityonion#13459] [2.4]
I am Olaitan, reaching out to seek clarification based on challenges I am having with my security onion (configuration :import node, version 2.4.6, Installation ISO, machine spec 8GB RAM, 512ROM, 4 CPU, have above minimum requirements).

Imported pcap from malware traffic analysis site.
I have configured and set up everything, I imported pcap and it's not showing any details nor alerts of the pcap.

I checked the grid and everything is running, status is ok

Please what is the way out 

Your response and contributions will be appreciated 

Thank you ![PXL_20240806_145241352.jpg](https://github.com/user-attachments/assets/e906bb16-2960-438c-a262-44ddbf16cecc)

**Comment by dougburks** [comment]
It looks like you've already created a discussion at https://github.com/Security-Onion-Solutions/securityonion/discussions/13442.

Please do not create multiple discussions for the same problem.
